Tallies/tallyAdmin_class.f90

```diff
@@ -17,6 +17,7 @@ module tallyAdmin_class
   ! Nuclear Data Interface
   use nuclearDataReg_mod,     only : ndReg_get => get
   use nuclearDatabase_inter,  only : nuclearDatabase
+  use materialMenu_mod,       only : mm_nMat => nMat
```

Comment:
As with the clerk, not sure if this needs to be here necessarily. A tally map fed in during initialisation should take care of this.